ingestion_tools/scripts/gjensen_config.py

```diff
@@ -105,6 +105,9 @@ def to_dataset_config(
         config["tissue"] = dataset["tissue"]
     if cross_reference:
         config["cross_references"] = cross_reference
+        if "dataset_publications" in config["cross_references"]:
+            config["cross_references"]["publications"] = config["cross_references"]["dataset_publications"]
+            del config["cross_references"]["dataset_publications"]
```

Comment:
Validation doesn't support dataset_publications, so we should probably keep it?

Reply:
Because of the changes [here](https://github.com/chanzuckerberg/cryoet-data-portal-backend/blob/db8268874e758627a56e09029226be38c71da296/ingestion_tools/scripts/gjensen_config.py#L385-L394), we might no longer need this.